cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -499,3 +499,454 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
+                * Fill Name and click Create
+            2. Test that snapshot can be created from VM Details page
+                * Navigate to My Services -> <service_name> -> <vm_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            3. Test that snapshot can be created from Snapshot page
+                * Navigate to My Services -> <service_name> -> <vm_name> -> Snapshots
+                * Click Configuration -> Create Snapshot
+                * Fill Name and click Create
+        expectedResults:
+            1. Snapshot successfully created
+            2. Snapshot successfully created
+            3. Snapshot successfully created
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_notification_for_snapshot_delete_failure_ec2():
+    """
+    Requires ec2 access via web-ui.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Provisioning
+        caseimportance: medium
+        initialEstimate: 1/4h
+        title: test notification for snapshot delete failure
+        setup:
+            1. Add ec2 provider
+        testSteps:
+            1. Create a snapshot on EC2 provider
+            2. Try to delete snapshot via CFME UI
+        expectedResults:
+            1. Snapshot created
+            2. Snapshot not deleted and notification displayed
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_snapshot_timeline_group_actions():
+    """
+    Test the SUI snapshot timeline.
+    Test grouping of actions in a timeline. Try to create a couple of
+    snapshots in a rapid succession, check how it looks in the timeline.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/3h
+        setup:
+            1. Add vmware provider
+        testSteps:
+            1. create a new vm
+            2. create multiple snapshots in fast succession (two should be enough)
+            3. go to the VM details page, then Monitoring -> Timelines
+            4. select "Management Events" and "Snapshot Activity" and click Apply
+            5. click on the group of events in timeline
+        expectedResults:
+            1. vm created
+            2. snapshots created
+            3. timelines page displayed
+            4. group of events displayed in the timeline
+            5. details of events displayed, correct number of events
+               displayed, time/date seems correct
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_snapshot_timeline_new_vm():
+    """
+    Test the SUI snapshot timeline.
+    See if there"s no timeline when there"s no snapshot.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/6h
+        setup:
+            1. Add vmware provider
```

Comment:
Test function should be marked only for vmware if that's the relevant provider for the test.

Reply:
Added markers; changed required providers.